### ISSUE TITLE
possible error in examples/utils.py

### ISSUE THREAD

**Issue by Dutchflag** [alexandrebarachant/muse-lsl#104]
i've been playing around with the neurofeedback example, and i noticed the following.

In line 85 of utils.py, the PSD is calculated. However, without taking the square of the magnitude of the FFT, you get the true theoretical FFT and not the power spectrum. I think the **2 is missing?

**Comment by jdpigeon** [comment]
Good catch. I think we can update that function soon to provide a true Welch's PSD implementation through scipy.

**Comment by oori** [comment]
> Welch's PSD implementation 

@jdpigeon, For reference code and explanation see [Compute the average bandpower of an EEG signal](https://raphaelvallat.com/bandpower.html) or his YASA library's [yasa.bandpower](https://raphaelvallat.com/yasa/build/html/generated/yasa.bandpower.html)  function
